### PR TITLE
Fix Authorization Header leak

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Resolves: BZ#2056113

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh, @GowthamShanmugam, @SanjalKatiyar Please review

File: package.json [hunk 1]
```diff
@@ -67,7 +67,6 @@
     "@types/react": "16.8.13",
     "@types/react-helmet": "^6.1.1",
     "@types/react-router-dom": "5.1.2",
-    "@types/webpack-dev-server": "^3.11.5",
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply package.json:70]: cmd: npm i --package-lock-only

**Comment by bipuladh** [comment]
Run `yarn install` to update yarn lock file as well.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply package.json:70]: Sure will do it

**Comment by TimothyAsirJeyasing** [comment]
done yarn install Please review @bipuladh , @GowthamShanmugam

**Comment by TimothyAsirJeyasing** [comment]
/retitile Bug 2056113: Fix Authorization Header leak

**Pull request merged**